plugins/rbac/src/components/CreateRole/AddMembersForm.tsx

```diff
@@ -73,7 +75,7 @@ export const AddMembersForm = ({
       <br />
       <Autocomplete
         options={membersOptions}
-        getOptionLabel={(option: SelectedMember) => option.label}
+        getOptionLabel={(option: SelectedMember) => option.label || ''}
```

Comment:
@debsmita1 We had this conversation regarding removing this here https://github.com/janus-idp/backstage-plugins/pull/1037#discussion_r1449893349. LMK if this was added back intentionally.